src/style.css

```diff
@@ -1 +1,29 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars.{
+  display: flex;
+}
```

Comment:
The selector `.stars.{` is incorrect. It should be `.stars` without the dot before the curly brace.